src/pages/accessibility/index.md

```diff
@@ -0,0 +1,147 @@
+
+At the beginning of phase 1, we ran accessibility reports on each document and assigned each document an owner. We sent each owner a list of the documents assigned to them for the phase along with the accessibility report. We also provided a link to our [wiki](https://github.com/ONRR/onrr.gov-site/wiki/Accessibility) with resources to make documents accessible. We also gave the owners a deadline about 8 weeks from the start date. One program area had the bulk of the documents, so we gave them a longer deadline of 20 weeks.
+
+We defined phase 2 as the next 100 most downloaded documents, which got us to 84% of downloaded documents. We sent each document owner their list of documents and accessibility reports. We gave document owners another 8 weeks to complete, phase 2, but it went faster than phase 1 because the document owners were now familiar with the making documents accessible.
+
+Then we had the long tail of phase 3. This phase included over 3,000 documents downloaded fewer than 100 times each in a year. To break up phase 3, I categorized the remaining documents.
+
+The bulk of the phase 3 documents were Federal Register Notices and public comments on those notices. Our regulations team had told us we could link out to the Federal Register website for the notices instead of hosting our own pdfs of them. To do this, we had our data intern find the appropriate links to the Federal Register and I got to work replacing those documents. That reduced the number of documents by 460. Finding links to replace the public comments with wasn't as easy, so the regulations team found us links and told us which we could remove. That allowed us to remove 1,288 more documents from the site.
+
+For the rest of phase 3, we wanted the document owners to decide whether to keep the documents before we ran accessibility reports. We gave the owners a list of documents along with how infrequently they were downloaded to make this decision.
+
+As I was going through the phase 3 documents, it became clear that there were about 2,100 extra documents in our website folder. We assumed that these documents were not linked on the site, since they didn't appear in the content inventory. Rather than spend time prioritizing these documents, I put them all in a "not linked" folder. We'll put them back if anyone complains about a broken link. Otherwise, we will not bring them to the new site.
```

Comment:
remove comma

```suggestion
As I was going through the phase 3 documents, it became clear that there were about 2,100 extra documents in our website folder. We assumed that these documents were not linked on the site since they didn't appear in the content inventory. Rather than spend time prioritizing these documents, I put them all in a "not linked" folder. We'll put them back if anyone complains about a broken link. Otherwise, we will not bring them to the new site.
```